### PR TITLE
proper upper bounding

### PR DESCRIPTION
**Pull request by xiaodaigh**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -18,11 +18,11 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 test = ["Test"]
 
 [compat]
-DataValues = "≥ 0.4.5"
-FileIO = "≥ 1.0.0"
+DataValues = "^0.4.13"
```
Comment (xiaodaigh) [inline Project.toml:21]:
```suggestion
DataValues = "^0.4.5"
```

**Comment by proudindiv** [comment]
So what do those of us that aren't developers do to fix Queryverse?

**Comment by xiaodaigh** [comment]
> So what do those of us that aren't developers do to fix Queryverse?
> I don't even use Parquet?

are you referring to #26?

Trying 

1. `]pin Parquet@0.4` and 
2. wait for ParquetFiles.jl to update and then 
3. `add Parquet`

**Comment by proudindiv** [comment]
Yes, thank you.

**Comment by davidanthoff** [comment]
Excellent, thanks!

I also opened https://github.com/JuliaRegistries/General/pull/15394, which should fix this for the existing tagged versions.

**Pull request merged**